source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ADCIOSViewController.mm

```diff
@@ -0,0 +1,127 @@
+//
+//  ADCIOS.m
+//  ADCIOS
+//
+//  Created by jwoo on 5/26/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "ADCIOSViewController.h"
+#import "ADCIOSAdaptiveHostConfig.h"
+#import "AdaptiveCard.h"
+#import "TextBlock.h"
+#import "Image.h"
+#import <AVFoundation/AVFoundation.h>
+
+using namespace AdaptiveCards;
+
+@implementation ADCIOSViewController
+std::shared_ptr<AdaptiveCard> adc;
```

Comment:
>adc [](start = 30, length = 3)

Is using acronym variable names the coding convention for objc? Feels like it becomes hard to read without proper names.